tests/core/fixtures.py

```diff
@@ -77,12 +83,70 @@ def chain(chaindb):
     return chain
 
 
+SHARD_CHAIN_CONTRACTS_FIXTURE = {
+    "contract_code": contract_bytecode,
+    "deployed_address": contract_address,
+    "initial_balance": 100000000,
+}
+
+
 @pytest.fixture
 def shard_chain():
     shard_chaindb = BaseChainDB(get_db_backend(), state_backend_class=FlatTrieBackend)
     return chain(shard_chaindb)
 
 
+@pytest.fixture
+def shard_chain_without_block_validation():
+    """
+    Return a Chain object containing just the genesis block.
+
+    This Chain does not perform any validation when importing new blocks.
+
+    The Chain's state includes one funded account which is where the simple transfer
+
+    contract will be deployed at.
+    """
+    # TODO: Comment this out to use NestedTrieBackend to prevent access list validation.
+    # Uncomment this to use FlatTrieBackend once access list generation is implemented.
+    # shard_chaindb = BaseChainDB(get_db_backend(), state_backend_class=FlatTrieBackend)
```

Comment:
I'm not sure if this is what you want, but assuming you want to run this using *both* state backends you can do that in a very clean way as follows.

```python
@pytest.fixture(params=(FlatTrieBackend, NestedTrieBackend))
def shard_chain_without_block_validation(request):
    shard_chaindb = BaseChainDB(get_db_backend(), state_backend_class=request.param)
    ...
```

Reply:
Sorry my comment was misleading. What I mean is to comment out FlatTrieBackend and use NestedTrieBackend instead for now. After the helper function which generates access list for a transaction is implemented, we can uncomment it and use FlatTrieBackend. I will update this part.